onnx_tool/node.py

```diff
@@ -463,6 +463,19 @@ def __init__(self, node_proto):
     def value_infer(self, intensors: []):
         y = max(0, min(1, self.alpha * intensors[0] + self.beta))
         return [y]
+    
+
+@NODE_REGISTRY.register()
+class HardSwishNode(PWNode):
+    def __init__(self, node_proto):
+        super().__init__(node_proto)
+        self.op_mac = MUL_MACS * 2 + ADD_MACS + CMP_MACS * 2
+        self.add_default_value('alpha',0.2)
```

Comment:
The default value of alpha should be 1/6, please refer to https://onnx.ai/onnx/operators/onnx__HardSwish.html